src/NexusMods.App/Startup.cs

```diff
@@ -59,6 +60,7 @@ public static void Main(IServiceProvider provider, string[] args)
         }
         catch (Exception e)
         {
+            Debugger.Break();
```

Comment:
Leftover from debugging I think

Reply:
```suggestion
```

Reply:
Nope, this is in here on purpose. Otherwise the debugger doesn't break.